venues/ICLR.cc/2020/Conference/python/buddy_ac.py

```diff
@@ -0,0 +1,133 @@
+                members = []
+            )
+        )
+
+        # Add paper's buddy AC as member to paper's AC group
+        paper_ac_group = map_paper_ac_groups[paper_number]
+        client.add_members_to_group(paper_ac_group, individual_buddy_group_id)
+
+        # Add paper's buddy AC as non-invitee for paper's meta-review invitation
+        paper_meta_rev_invitation = map_meta_review_invitations[paper_number]
+        paper_meta_rev_invitation.noninvitees
+
+        ## Post AC & AC-Buddy only Comment invitation for this paper
+        ac_conversation_invitation = openreview.Invitation(
+            id = conference_id + '/Paper' + paper_number + '/-/Area_Chair_Only_Comment',
+            readers = [
+                conference.get_area_chairs_id(number = paper_number),
+                individual_buddy_group_id,
+                conference_id
+            ],
+            writers = [conference_id],
+            signatures = [conference_id],
+            invitees = [
+                conference.get_area_chairs_id(number = paper_number), individual_buddy_group_id
+            ],
+            reply = {
+                "forum" : paper.forum,
+                "readers": {
+                    "description": "Select all user groups that should be able to read this comment.",
+                    "values": [
+                        "ICLR.cc/2020/Conference/Paper" + paper_number + "/Area_Chairs",
+                        "ICLR.cc/2020/Conference/Paper" + paper_number + "/Buddy_Area_Chair1",
+                        conference.get_program_chairs_id()
+                    ]
+                    },
+                "writers": {
+                    "values-regex": "ICLR.cc/2020/Conference/Paper6/(Buddy_)*Area_Chair[0-9]+",
+                    "description": "How your identity will be displayed."
+                },
+                "signatures": {
+                    "values-regex": "ICLR.cc/2020/Conference/Paper6/(Buddy_)*Area_Chair[0-9]+",
+                    "description": "How your identity will be displayed."
+                },
+                "content": {
+                    "comment": {
+                        "value-regex": "[\\S\\s]{1,5000}",
+                        "required": True,
+                        "order": 1,
+                        "description": "Your comment or reply (max 5000 characters)."
+                    },
+                    "title": {
+                        "value-regex": ".{1,500}",
+                        "required": True,
+                        "order": 0,
+                        "description": "Brief summary of your comment."
+                    }
+                }
+            },
+            process = 'buddyAcCommentProcess.js'
+        )
+        client.post_invitation(ac_conversation_invitation)
+
+    all_anon_revs = openreview.tools.iterget_groups(client, regex = 'ICLR.cc/2020/Conference/Paper[0-9]+/AnonReviewer[0-9]+')
+    for anon_rev in all_anon_revs:
```

Comment:
I'm not sure if you have to do this, using PaperX/Area_Chairs should be enough.

Reply:
Agreed. Fixed.